PYME/LMVis/Extras/pointSetGeneration.py

```diff
@@ -139,37 +170,83 @@ def refresh_choices(self):
 
 
 
-
-
 class Generator(HasTraits):
     meanIntensity = Float(500)
     meanDuration = Float(3)
     backgroundIntensity = Float(300)
     meanEventNumber = Float(2)
     scaleFactor = Float(2)
     meanTime= Float(2000)
+    mode = Enum(['STORM','PAINT'])
 
     sources = List([WormlikeSource(), ImageSource(), FileSource()])
 
     source = Instance(PointSource)
-    
+
+    helpInfo = {
+        'source' : '''
+Select the type of point source to generate points.
+A wormlike source, an image based source and a file based source are supported.
+''',
+        'meanIntensity' : '''
+This parameter specifies the mean number of photons in an event.
+Typically values are in the range from 100 to several 10000.
+''',
+        'meanDuration' : '''
+The mean duration of events which is specified in units of frames.
+''',
+        'meanTime' : '''
+This parameter, the mean time of the series in frame units, is related to the series duration.
+For PAINT mode this is half the duration of the series, for STORM simulation mode the relationship is a little more complex.
```

Comment:
in both cases, `meanTime` is the average time at which you expect to get events (i.e. target value of `np.mean(pipeline['t'])`) . STORM mode draws event time from an exponential distribution, PAINT from a uniform one.

Reply:
I will add a sentence to this end. Note that the typical user will be an experimenter to whom the concept of a mean time may mean very little. So it will be helpful to give something related to what they know about as well, which IMHO is series duration.